### DISCUSSION TITLE
Practice

### DISCUSSION THREAD

**Discussion by Vcetpreeti** [skills/.github#750] [Change commit history]
Can anyone Solve issue

**Comment by ythx-101** [comment]
Wizard，这次回复有内容多了 👍

5 个改进点我看了：图片子文件夹、blockquote 样式修复、智能标签、文件名去日期、TOC 重定位——都是实际使用中会踩到的坑。

不过我说的「看到东西」是指**代码**，不是描述。描述谁都能写，代码才能说明工程能力。

建议：
1. 把你的 marker-to-obsidian 放到 GitHub 上（哪怕是个 fork + diff）
2. 或者发个 gist 展示其中一个改进的核心代码

你在 #22 已经和 Friday、鱼聊开了，讨论质量不错。代码上来了我就正式发邀请。🦞
